src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -185,5 +185,36 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 
 		snapshot("screenshot_certificate_overview_vaccination_and_test_certificate")
 	}
+	
+	func test_screenshot_MultipleFamilyTestCertificates() throws {
+		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
+		app.setLaunchArgument(LaunchArguments.healthCertificate.familyCertificate, to: true)
+		app.launch()
+
+		// Navigate to Certificates Tab.
+		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap()
+		
+		var healthCertificateCell = app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.healthCertifiedPersonCell]
+		XCTAssertTrue(healthCertificateCell.waitForExistence(timeout: .short))
+		snapshot("screenshot_certificate_family_certificats-cert-1")
+		
+		app.swipeUp(velocity: .slow)
+		
+		healthCertificateCell = app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.healthCertifiedPersonCell]
```

Comment:
you are checking the same cell multiple times, you can count the number of cells

Reply:
Sure let me just think about for a second :

Reply:
I just refactored it

Reply:
I can not see the changes here.

Reply:
But isn't it the same cell but 4 times now. So this is why I am checking this.

Reply:
Cells is an array, if there is one such cell, it will be true. Please check for the count, it should be 4. Otherwise, the test is not useful.

Reply:
Added the additional count check.

Reply:
<3